src/twisted/python/_release.py

```diff
@@ -39,14 +40,66 @@
 
 
 def runCommand(args, **kwargs):
-    """Execute a vector of arguments.
+    """
+    Execute a vector of arguments.
 
     This is a wrapper around L{subprocess.check_output}, so it takes
     the same arguments as L{subprocess.Popen} with one difference: all
     arguments after the vector must be keyword arguments.
+
+    @param args: arguments passed to L{subprocess.check_output}
+    @param kwargs: keyword arguments passed to L{subprocess.check_output}
+    @return: command output
+    @rtype: L{str}
     """
     kwargs['stderr'] = STDOUT
-    return check_output(args, **kwargs)
+    output = check_output(args, **kwargs)
+
+    # The check_output() function always returns bytes
+    # on Python 2 and 3, but we want this function to always return
+    # str.
```

Comment:
The code in this module is expecting the output of this function to be
`str` or _native string_.  I found it easier to make this change here rather than change
all the places that use output from this